lib/services/userService.dart

```diff
@@ -179,25 +179,14 @@ Future<String> getScopePermissions() async {
   return prefs.getString('scopePermissions');
 }
 
-Future<bool> clearData({context}) async {
+Future<void> clearData({context}) async {
   final prefs = await SharedPreferences.getInstance();
 
-  Response response;
-
-  // Incase we are not connected to the internet or the DNS fails to resolve.
   try {
-    response = await removeDeviceId(prefs.getString('doubleName'));
+    await removeDeviceId(prefs.getString('doubleName'));
   } catch (e) {
     print(e);
-    response = null;
   }
 
-  if (response != null && response.statusCode == 200) {
-    print("Removing account");
-    prefs.clear();
-    return true;
-  } else {
-    print("Something went wrong while removing your account");
-    return false;
-  }
+  prefs.clear();
```

Comment:
This means that your deviceId will remain on the backend if you delete your account from your phone without internet connection. Are we sure we want this?

Reply:
Yeah, I've thought about it. Device ID by itself is harmless and will be overwritten anyways if the user recovers their account. So one could argue we don't even need to remove the device ID.

Reply:
You know why we did this right?
We did this so people with a new account on their device would not get a notification from their old account.
Also when the deviceId is removed from the backend we know we can't send a notification and should prompt the user (via the frontend) to account recovery instructions.

Reply:
@MathiasDeWeerdt I think @azertyalex has a point.
This call is needed because now a user can still get notifications on their phone if they've removed the account from their device but (s)he wouldn't be able to login

Reply:
I'll reject this as discussed on Telegram :)